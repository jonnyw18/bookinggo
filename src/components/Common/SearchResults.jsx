import React from 'react';

const SearchResults = ({searchData, showResults}) =>
    <div hidden={showResults} className="test-scss">
        <ul>
            {
                searchData.length > 0
                    ? searchData.map((locations, index) => (
                        locations.city
                            ? <div key={index}>
                                <li> {locations.city} </li>
                              </div>
                            : <div key={index}>
                                <li key={index}> {locations.name} </li>
                              </div>
                    ))
                    : null
            }
         </ul>
    </div>;

export default SearchResults