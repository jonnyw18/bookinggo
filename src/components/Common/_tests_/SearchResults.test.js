import SearchResults from "../SearchResults";

describe('<SearchResults />', () => {
    it('renders correctly', () => {
        const wrapper = shallow(
            <SearchResults
                searchData={[]}
                hideResults={true}
                getSelectedLocation={() => {}}
            />
        );
        expect(wrapper).toMatchSnapshot();
    });
});
