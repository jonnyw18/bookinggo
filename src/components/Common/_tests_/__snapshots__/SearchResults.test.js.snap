// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchResults /> renders correctly 1`] = `
<div
  className="search-results-container"
  hidden={true}
>
  <ul />
</div>
`;
